src/Hidden/HiddenJs.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+/**
+ * Responsively hides children by omission.
+ */
+import React, { Element } from 'react';
+import { keys as breakpoints } from '../styles/breakpoints';
```

Comment:
Can we get those values through the theme context? That would make them configurable, better 👍 .